heat/files/mitaka/heat.conf.Debian

```diff
@@ -445,8 +445,8 @@ auth_url = http://{{ server.identity.host }}:35357
 username = {{ server.identity.user }}
 password = {{ server.identity.password }}
 user_domain_name = default
-project_domain_id=Default
-user_domain_id=Default
+project_domain_id = {{ controller.identity.get('domain', 'default') }}
```

Comment:
I just copy it from other formula, it is the same in nova, glance, neutron, etc.
See here for example: https://github.com/salt-formulas/salt-formula-nova/blob/master/nova/files/mitaka/nova-controller.conf.Debian#L138-L139